cmd/doc.go

```diff
@@ -0,0 +1,5 @@
+// Package cmd stores the command line interface implementation.
+//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+package cmd
```

Comment:
It should be of the following order

i) Copyright License Header
ii) Package documentation

Reply:
erg, right, I took example on an existing package in aks-engine, which was wrong to begin with :)

Reply:
I'm not sure that's true. We should run `godoc` as a server and make sure that the comment is picked up, because I had a reason for committing the initial comments that way.